_pages/en/strategy-build-vs-buy.md

```diff
@@ -56,20 +56,20 @@ permalink: /strategy-build-vs-buy.html
 
 ### Purpose
 
-To provide IITB with a build versus buy assessment framework.
+To provide IITB with a Build Versus Buy Assessment Framework.
 
 The strategy includes:
 
-1. A **guiding policy**, which serves to set automatic decisions that defines the build versus buy assessment framework
-2. A **coherent set of actions** (an action plan), which serve to operationalize the build versus buy assessment framework.
+1. A **guiding policy**, which serves to set automatic decisions that defines the Build Versus Buy Assessment Framework
+2. A **coherent set of actions** (an action plan), which serve to operationalize the Build Versus Buy Assessment Framework.
 
 The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
 
 ### Target Audience
 
-This **strategy document** is targeted to stakeholders involved in determining whether to build or to buy software components.
-More specifically, stakeholders involved in architecting solutions, defining technology standards, developing, delivering, operationalizing and maintaining IT Solutions (such as Enterprise Architecture, Enterprise Operations, etc.) and guiding committees such as Enterprise Architecture Review Board.
-This includes both IITB and non-IITB stakeholders (such as from Business Functions, SSPB, CDO, ISB, IAERMB, and SSC).
+This **strategy document** is targeted to stakeholders involved in determining whether to build or to buy IT Solutions and Products.
+More specifically, stakeholders involved in architecting IT solutions, defining technology standards, developing, delivering, operationalizing and maintaining IT Solutions (such as Enterprise Architecture, Enterprise Operations, etc.) and guiding committees such as the Architecture Review Committee and the Enterprise Architecture Review Board.
+This includes both IITB and non-IITB stakeholders (such as Business Functions, SSPB, CDO, ISB, IAERMB, and SSC).
 The list of stakeholders are listed in section [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
 
 The **Guiding Policy**, once operationalized, will target stakeholders involved when the need for an IT Solution is raised, when the IT Solution is conceptualized, architected, developed, delivered, operationalized, and maintained.
```

Comment:
> when the IT Solution 

should be "where an IT solution ..."

Reply:
The sentence was with regards as to "when" the rules apply. I've modified the syntax to better reflect the nuance.